Source/WebCore/css/calc/CSSCalcTree+Parser.cpp

```diff
@@ -84,8 +85,7 @@ struct ParserState {
 
 static ParseStatus checkDepthAndIndex(int depth, CSSParserTokenRange tokens)
 {
-    if (tokens.atEnd())
-        return ParseStatus::NoMoreTokens;
+    UNUSED_PARAM(tokens);
```

Comment:
`anchor-size()` is a valid configuration, so it's possible the token range is empty.

Reply:
All tests are passing, so doesn't seem this would affect correctness of anything.